sys/kern/vfs.c

```diff
@@ -341,6 +341,29 @@ int vfs_lookup(const char *path, vnode_t **vp) {
   return error;
 }
 
+int vnr_create(const char *path, vnode_t **dvp, componentname_t *cn) {
```

Comment:
I guess we should rename it to `vfs_namecreate` and `vfs_lookup` to `vfs_namelookup` (possibly in another PR).

Reply:
`vfs_lookup` renamed in another PR.